src/rtree_intf.ml

```diff
@@ -77,6 +89,9 @@ module type Envelope = sig
 
   val contains : t -> t -> bool
   (** [contains a b] asks whether [b] is contained by [a]. *)
+
+  val minmaxdist : float list -> t -> float
```

Comment:
Why are these needed in envelope?

Reply:
We could put this in a separate module called Point, or create a function in an envelope, to calculate the distance from a point to that envelope. This seemed easier to me. I know this is not the appropriate place for the function. Need to find its home.